src/styl/_components/breakingnews.twig

```diff
@@ -1,5 +1,11 @@
-<a href="#" class="breakingnews box">
-    <span class="breakingnews-label">Breaking News</span>
-    <strong class="breakingnews-title">Important title of a new important article</strong>
-    <span class="breakingnews-cta">Read this article</span>
-</a>
+<section class="breakingnews" >
+    <div class="breakingnews-content">
+        <a class="d-flex ai-cntr" href="#">
```

Comment:
It's odd to have helpers here. Especially if the children have props that depends on it. Use a class here add its related style